SECURITY.md

```diff
@@ -12,4 +12,4 @@ The versions of the project that are currently supported with security updates.
 
 ## Reporting a Vulnerability
 
-You can report a vulnerability following the instructions in https://tidelift.com/docs/security.
+You can report a vulnerability sending us an email at material.ui.org@gmail.com.
```

Comment:
Is it possible that we can tie this to our docs domain i.e. `security@material-ui.com`? Can work on a better address later. This is good for merge.

Also: Please forward emails to that address to me as well.

Reply:
💯 on that!